_posts/2025-09-19-minimizing-dependencies.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "The importance of minimizing dependencies in Bitcoin Core"
+permalink: /blog/2025/09/19/minimizing-dependencies/
+layout: post
+author: brink
+name: "brink"
+alt: The importance of minimizing dependencies in Bitcoin Core
+category: "technical"
+description: Relying on external code is a risk. We look at the work in Bitcoin Core to minimize these kind of dependencies.
+---
+
+The security of Bitcoin Core's codebase is rightfully emphasized in discussions
+of the Bitcoin network's resiliency. But Bitcoin Core's code is not the only
+code executed when running Bitcoin Core. Over the years, Bitcoin Core has also
+relied on external, third-party code and libraries. Some of these dependencies
+have been the source of historical vulnerabilities, and all of them carry
```

Comment:
> source of historical vulnerabilities

Could link to https://bitcoincore.org/en/2024/07/31/disclose-upnp-oom/ or https://bitcoincore.org/en/2024/07/03/disclose_upnp_rce/.